### PR TITLE
Remove old parameter "post_behavior" and "failure_post_behavior"

### PR DESCRIPTION
**Pull request by aleksbykov**
Remove old parameters  "post_behavior" and "failure_post_behavior" and logic which  uses it from code. Start using only new parameters to control resources:
post_behavior_db_nodes
post_behavior_loader_nodes
post_behavior_monitor_nodes

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [ ] I didn't leave commented-out/debugging code
- [ ] I didn't copy-paste code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [x] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by roydahan** [comment]
There are more PRs pending with the old behaviour.
Let's keep this one for them as well and update this PR as soon as they get merge.
(soon).

File: sdcm/tester.py [hunk 1]
```diff
@@ -1426,39 +1430,41 @@ def clean_resources(self):
...
+                cluster.Setup.keep_cluster(node_type='monitor_nodes', val='keep')
+                self.db_cluster.set_keep_tag_on_failure()
 
         if self.credentials is not None:
             cluster.remove_cred_from_cleanup(self.credentials)
-            if self._failure_post_behavior == 'destroy':
-                for credential in self.credentials:
-                    credential.destroy()
-                self.credentials = []
+            # if self._failure_post_behavior == 'destroy':
```
Comment (fruch) [inline sdcm/tester.py]: please remove comment

File: vars/longevityPipeline.groovy [hunk 2]
```diff
@@ -85,7 +91,9 @@ def call(Map pipelineParams) {
                                         exit 1
                                     fi
 
-                                    export SCT_FAILURE_POST_BEHAVIOR="${pipelineParams.params.get('post_behaviour', '')}"
+                                    export SCT_POST_BEHAVIOR_DB_NODES="${pipelineParams.params.get('post_behaviour_db_nodes', '')}"
```
Comment (fruch) [inline vars/longevityPipeline.groovy]: you need to go over all pipelines and update this 
i.e. lookup 'SCT_FAILURE_POST_BEHAVIOR' it's there in all pipelines

File: sdcm/tester.py (hunk 1 above)
Comment (aleksbykov) [reply sdcm/tester.py]: Fixed

File: vars/longevityPipeline.groovy (hunk 2 above)
Comment (aleksbykov) [reply vars/longevityPipeline.groovy]: Fixed

**Comment by fruch** [comment]
@aleksbykov, seem like you still have pre-commit issues on this PR, please run it manually:
`pre-commit run -a` to fix the issue

**Comment by bentsi** [comment]
@aleksbykov please address Israel's comment and when rebased check that precommit succeeds

File: sdcm/utils/common.py [hunk 3]
```diff
@@ -1223,8 +1223,7 @@ def get_post_behavior_actions(config):
 
     for key in action_per_type:
         config_key = 'post_behavior_{}'.format(key)
-        old_config_key = config.get('failure_post_behavior', 'destroy')
-        action_per_type[key] = config.get(config_key, old_config_key)
+        action_per_type[key] = config.get(config_key, 'destroy')
```
Comment (bentsi) [inline sdcm/utils/common.py]: defaults should be in one place, that is the test_default.yaml

Comment (aleksbykov) [reply sdcm/utils/common.py]: test_default.yaml already has appropriate settings. Removed unused defauls

**Pull request merged**